client/my-sites/media-library/index.jsx

```diff
@@ -141,11 +148,7 @@ class MediaLibrary extends Component {
 				return ! ( ( site && site.options.upgraded_filetypes_enabled ) || isJetpack );
 
 			case 'videos':
-				return ! (
-					( site && site.options.videopress_enabled ) ||
-					( isJetpack && ! isAtomic ) ||
-					( isAtomic && hasVideoUploadFeature )
-				);
+				return ! hasVideoUploadFeature && !! hasVideoUploadAvailableFeature;
```

Comment:
I think this is the most relevant change here, replacing the site type/platform approach with the features one. We don't need to check for Simple, Atomic, or Jetpack sites.

Reply:
I much prefer this approach, but I'm not sure if this logic is exactly the same. It could be but I'm not sure, as it all depends on the logic on the server. We might want to add some tests as part of D63248-code.

Reply:
After talking about this approach, we agreed to add another `video-hosting` feature. However, we don't need it on this PR. `upload-video-files` will determine whether the site has or hasn't the ability to upload a video file.
Let me mark this comment as resolved.